pkg/cli/preview/preview_test.go

```diff
@@ -34,6 +34,28 @@ import (
 
 // TestPreview creates an object using KCC, and then runs the preview command to look for additional changes
 func TestPreview(t *testing.T) {
+	var testResources map[string]string = map[string]string{
```

Comment:
It is probably easier to treat this as a table-driven test, and to t.Run each: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

When the number of tests becomes too large, we typically move them to the filesystem (under `testdata/`).  But 2 is very manageable :-)

But I think you want to check each test separately...